Sources/ProjectSpec/Dependency.swift

```diff
@@ -36,7 +36,7 @@ public struct Dependency: Equatable {
     public enum DependencyType: Equatable {
         case target
         case framework
-        case carthage(findFrameworks: Bool?)
+        case carthage(findFrameworks: Bool?, static: Bool)
```

Comment:
@yonaskolb If we required Swift 5.1 to compile we could use a default value here instead of having to have `static: false` added everywhere.

Reply:
CI seems to be still run on Xcode 10. https://github.com/yonaskolb/XcodeGen/blob/03b74b37c0b7659b7c6de8ad4d6e69e4491391a1/.github/workflows/ci.yml#L11

So I maintained backward compatibility.

Reply:
CI environments for legacy Swift versions are dropped on #709.

So I used default associated value. 
https://github.com/yonaskolb/XcodeGen/pull/688/commits/75905237dee8aea274fbdecb2a52b77c7689055d